research/sentiment_analysis/data/util.py

```diff
@@ -0,0 +1,26 @@
+import tensorflow as tf
+import numpy as np
+
+START_CHAR = 1
+END_CHAR = 2
+OOV_CHAR = 3
+
+
+def pad_sentence(sen, sentence_length):
+  sen = sen[:sentence_length]
+  if len(sen) < sentence_length:
+    sen = np.pad(sen, (0, sentence_length - len(sen)), 'constant',
```

Comment:
I am not sure where is my existing comment, but from API, you are padding 0 at the sentence start, which means START_CHAR is not used at all. Is this intended?

Reply:
START_CHAR is used when loading the dataset from keras module,
e.g. start_char=START_CHAR is a parameter.